shared/js/background/message-handlers.js

```diff
@@ -323,9 +279,6 @@ export function addDebugFlag(message, sender, req) {
  * @param {(options: any, sender: any, req: any) => any} func
  */
 export function registerMessageHandler(name, func) {
-    if (messageHandlers[name]) {
```

Comment:
How come we're removing this check?

Reply:
This was being problematic in tests, where creating a component in subsequent test cases threw an exception and failed. If we're worried about naming collisions in message handlers, we could just log here.